docs/navigation-configuration.md

```diff
@@ -1,242 +1,242 @@
-                pathSegment: 'pr1',
-                label: 'Project one',
-                viewUrl: 'https://my.microfrontend.com/projects/details.html#id=pr1',
-                hideSideNav: true
+````javascript 
+navigation: {
+  nodes: [
+    {
+      pathSegment: 'TopNav1',
+      label: 'Top Navigation Element One',
+      viewUrl: 'https://microfronted.com',
+      children: [
+        {
+          pathSegment: 'SideNav1',
+          label: 'Side Navigation Element One',
+          viewUrl: 'https://microfrontend.com/projects/list.html',
+          children: [
+            {
+              link: '/TopNav1/internalLink',
+              label: 'This takes you to yourwebsite.com/TopNav1/internalLink',
+            },
+            {
+              externalLink: {
+                url: 'http://www.google.com',
+                sameWindow: false
               },
-              {
-                pathSegment: 'pr2',
-                label: 'Project two',
-                viewUrl: 'https://my.microfrontend.com/projects/details.html#id=pr2'
-              }
-            ]
-          },
-          {
-            category: {label:'Misc', icon:'miscellaneous'},
-            pathSegment: 'miscellaneous',
-            label: 'Miscellaneous',
-            viewUrl: 'https://my.microfrontend.com/general/miscellaneous.html'
-          },
-          {
-            category:'Misc',
-            pathSegment: 'miscellaneous2',
-            label: 'Miscellaneous2',
-            viewUrl: 'https://my.microfrontend.com/general/miscellaneous2.html'
-          }
-        ]
-      }
-    ]
-  }
-}
+              label: 'This takes you to an external page',
+            },
+          ]
+        },
+      ]
+    },
+    {
+      pathSegment: 'TopNav2',
+      label: 'Top Navigation Element Two',
+      viewUrl: 'https://2ndmicrofronted.com',
+      children: [
+...
 ````
 
+## Basic navigation parameters
 
-## Node navigation
+>**NOTE:** For a full list of available parameters, see the [parameter reference](navigation-parameters-reference.md) document.
 
-When you navigate between nodes that are located in the same domain, Luigi triggers a hash or path change. Then it sends the updated context in order not to fully reload the view for a single-page application based micro frontend. Navigation between domains triggers a full page load in order to comply with cross-domain security concepts.
+In the "First steps" example, you saw some basic navigation parameters:
 
-If you start navigating from a child node level, navigate to the specific product route using the [Luigi Client API](luigi-client-api.md) as shown in the example:
+### pathSegment
+This is the path in the browser URL. The main application path is built from values in the navigation path, joined with the **/** character. For example, if the value of this parameter is `home`, the path will be `yourwebsite.com/home`. You can override this setting by using one of the following instead of **pathSegment**: 
```

Comment:
"This is the path in the browser URL." -  pathSegment is rather used to build the path. A particular pathSegment is one segment on the path. (I know this is clear but 'pathSegment - "This is the path in the browser URL." can be misleading.)